zstd-decompress.c

```diff
@@ -17,6 +14,7 @@
 #include <unistd.h>
 #include <zstd.h>
 
+#ifdef HAVE_ZSTD
```

Comment:
Why? zstd is currently a requirement, so it might be better to drop HAVE_ZSTD at all.